test/browser_tests/conf.js

```diff
@@ -121,25 +125,18 @@ var config = {
 
   getMultiCapabilities: function() {
     var params = this.params;
-    var capabilities = defaultSuites.essential;
 
-    // If --sauce=true flag was passed, configure Sauce Labs credentials.
-    if ( ( typeof params.sauce === 'undefined' || params.sauce === 'true' ) &&
-         _isSauceCredentialSet() ) {
-      capabilities = defaultSuites.full;
-    } else {
+    // If Sauce Labs credentials or --sauce flag is not set or is not true,
+    // delete Sauce credentials on the config object.
+    if ( !_isSauceCredentialSet() || params.sauce === 'false' ) {
       delete config.sauceSeleniumAddress;
       delete config.sauceUser;
       delete config.sauceKey;
     }
 
-    var newCapabilities = _copyParameters( params,
-                                           _chooseSuite( params,
-                                                         capabilities ) );
-    if ( newCapabilities.length !== 0 ) {
-      capabilities = newCapabilities;
-    }
-
+    var capabilities = _copyParameters( params,
+                                        _chooseSuite( params,
+                                                      capabilities ) );
     return capabilities;
```

Comment:
Moved `_chooseSuite` reference to its own variable in https://github.com/cfpb/cfgov-refresh/commit/29f9a222b69f307c01776769d008c26c75e95b2d